apps/system/test/unit/dialer_agent_test.js

```diff
@@ -219,8 +219,8 @@ suite('system/DialerAgent', function() {
         callschanged();
       });
 
-      test('it should not play the ringtone', function() {
-        assert.isTrue(mockAudio.play.notCalled);
+      test('it should play the silent ringtone', function() {
+        assert.isTrue(mockAudio.play.calledOnce);
```

Comment:
nit: You can use the new format for sinon assertions which is usually more terse
```javascript
sinon.assert.calledOnce(mockAudio.play)
```